### ISSUE TITLE
pip install vaex fails on M1 with Big Sur 11.2.3

### ISSUE THREAD

**Issue by InterruptSpeed** [vaexio/vaex#1246]
**Description**
%pip3 install vaex

results in error under Big Sir 11.2.3 on M1 processor [don't know if that matters, don't have another one test]

**Software information**
% pip3 --version
pip 21.0.1 from /usr/local/lib/python3.9/site-packages/pip (python 3.9)
% python3.9 --version
Python 3.9.2

**Additional information**
error output:
...
   running build_ext
    building 'vaex.vaexfast' extension
    creating build/temp.macosx-11-x86_64-3.9
    creating build/temp.macosx-11-x86_64-3.9/src
    clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -I/usr/local/lib/python3.9/site-packages/numpy/core/include -I/usr/local/include -I/usr/local/opt/openssl@1.1/include -I/usr/local/opt/sqlite/include -I/usr/local/opt/tcl-tk/include -I/usr/local/Cellar/python@3.9/3.9.2_1/Frameworks/Python.framework/Versions/3.9/include/python3.9 -c src/vaexfast.cpp -o build/temp.macosx-11-x86_64-3.9/src/vaexfast.o -std=c++11 -mfpmath=sse -O3 -funroll-loops -g -mmacosx-version-min=10.9
    error: $MACOSX_DEPLOYMENT_TARGET mismatch: now "10.9" but "11" during configure
    ----------------------------------------
ERROR: Command errored out with exit status 1
...

**Comment by GiuseppeLaurenza** [comment]
Hi, I found a workaround exporting MACOSX_DEPLOYMENT_TARGET="11" before launching the compilation.

**Comment by InterruptSpeed** [comment]
confirms that this works! you must also have this installed:

xcode-select --install

before doing pip install vaex.

thanks!

**Comment by maartenbreddels** [comment]
Thanks for the report,

FYI, I'm working on the binary wheels here: https://github.com/vaexio/vaex/pull/1259/checks

and the reason for this I think it because of these lines https://github.com/vaexio/vaex/blob/master/packages/vaex-core/setup.py#L33 which was changed in https://github.com/vaexio/vaex/pull/195
I wonder if I change that to "11" to make it work out of the box.

**Comment by InterruptSpeed** [comment]
happy to make a minor contribution. put in a pull request for the docs in the meantime [linked above]. thx folks.

**Comment by maartenbreddels** [comment]
I've build an arm64 version now, could you try installing this and let me know it works:
```
python -m pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple vaex-core==4.0.1.post7
```

**Comment by InterruptSpeed** [comment]
looks good to me. first i did:
% python3.9 -m virtualenv venv
% source venv/source/activate

then i ran that command which was very happy

then dropping into:
% python
can also
>>> import vaex

with no complaints!

all on m1 with big sur 11.2.3

**Comment by maartenbreddels** [comment]
Thanks!

Could you run this final check:
```
$ python -c "import vaex; print(vaex.example().describe())"
```

**Comment by maartenbreddels** [comment]
vaex-hdf5 is a dependency of vaex, but not vaex-core, so if you install vaex, you will get that.
Note that this is now released as part of vaex/vaex-core 4.1.0!
